docs/src/main/java/io/helidon/docs/se/openapi/OpenApiGeneratorSnippets.java

```diff
@@ -0,0 +1,388 @@
+/*
+ * Copyright (c) 2024 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.docs.se.openapi;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.helidon.http.Status;
+import io.helidon.webclient.api.HttpClientResponse;
+import io.helidon.webclient.api.WebClient;
+import io.helidon.webclient.api.WebClientConfig;
+import io.helidon.webserver.http.ServerRequest;
+import io.helidon.webserver.http.ServerResponse;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class OpenApiGeneratorSnippets {
+
+    /*
```

Comment:
See the full file [here](https://gist.github.com/romain-grecourt/f6d92cd092a27b49e5f2834c59e1f150).
The content can be pasted in a terminal directly.

I couldn't get `--helidonVersion` to work, but `-p helidonVersion=xxx` seemed to work.

```java
    /*
        # Instructions to generate a project containing the types which need to be mocked.
        # In case of changes in the upstream generator.
        # See https://github.com/OpenAPITools/openapi-generator
        # download the openapi-generator cli
        curl -O \
            --output-dir /tmp \
            https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/7.7.0/openapi-generator-cli-7.7.0.jar
        # generate the petapi server project (replace helidonVersion accordingly)
        java -jar /tmp/openapi-generator-cli-7.7.0.jar generate \
           -o /tmp/petapi-server \
           -g java-helidon-server \
           --library se \
           -i etc/petstorex.yaml \
           -p useAbstractClass=true \
           -p helidonVersion=4
        # generate the petapi client project (replace helidonVersion accordingly)
        java -jar /tmp/openapi-generator-cli-7.7.0.jar generate \
           -o /tmp/petapi-client \
           -g java-helidon-client \
           --library se \
           -i etc/petstorex.yaml \
           -p helidonVersion=4
    */
```

Reply:
Updated. My bad on the `--helidonVersion` typo. 

I did not hard-code the generator or Helidon versions into the shell commands.